plugins/analytics/public/malshare.py

```diff
@@ -35,7 +35,7 @@ def fetch(observable, api_key):
                 'api_key': api_key,
                 'action': 'details'
             }
-            response = requests.get('https://malshare.com/api.php', params)
+            response = requests.get('https://malshare.com/api.php', params, verify=False)
             if response.ok:
                 return response.json()
             else:
```

Comment:
We should probably raise an exception if something goes wrong here: exceptions are caught and displayed in red in the UI to alert the user that something is wrong with the feed.

Reply:
check now if that `raise` is corrent

Reply:
Aaah yeah that would work, but I would raise a `GenericYetiError` with some nicer message e.g. "Could not retrieve feed, HTTP response: {error_code}`